.readthedocs.yml

```diff
@@ -9,16 +9,10 @@ version: 2
 sphinx:
   configuration: docs/sphinx/source/conf.py
 
-# Build documentation with MkDocs
-#mkdocs:
-#  configuration: mkdocs.yml
-
-# Optionally build your docs in additional formats such as PDF
-formats:
-  - pdf
-
-# Optionally set the version of Python and requirements required to build your docs
-python:
-  version: 3.7
-  install:
-    - requirements: docs/sphinx/source/requirements.txt
+build:
+  os: "ubuntu-22.04"
+  commands:
+    - mkdir --parents $READTHEDOCS_OUTPUT/html/
+    - wget https://github.com/sofa-framework/SofaPython3/releases/download/release-v23.06/SofaPython3_v23.06_docs.zip
```

Comment:
v23.06?

Reply:
Yes, the doc was done at that time and as we have to manually update things I would prefer to first merge something that works then update the build process to remove the manual parts.